lib/libutee/tee_api_operations.c

```diff
@@ -1418,11 +1420,6 @@ TEE_Result TEE_AEEncryptFinal(TEE_OperationHandle operation,
 		goto out;
 	}
 
-	if (!srcData && !srcLen) {
-		res = TEE_SUCCESS;
-		goto out;
-	}
-
```

Comment:
Update: the C standards does require `src` to be valid even though `l` may be zero.